composer.json

```diff
@@ -11,10 +11,12 @@
             "email": "dcramer@gmail.com"
         }
     ],
+    "minimum-stability": "dev",
     "require-dev": {
         "friendsofphp/php-cs-fixer": "~2.1",
         "phpunit/phpunit": "^4.8 || ^5.0",
-        "monolog/monolog": "*"
+        "nokitakaze/testhttpserver": "^0.1.0",
```

Comment:
I honestly don't know what Tornado is, and I don't understand the point of the unit test added above that even uses a fake SSL certificate (or something like that). As I said, we don't have to test what a server replies, we just have to test that the server replies with something. Plus, we agreed that in the future we will use HttpPlug or a library like Guzzle to handle the sending of events (and this means that we won't have to make unit tests about async/not async because it won't anymore be our concern), so I don't really get how this could be useful at all if we're going to remove all the current cURL code